sonic.asm

```diff
@@ -1119,22 +1119,24 @@ TilemapToVRAM:
 
 		include	"_inc\Nemesis Decompression.asm"
 
+
+; ---------------------------------------------------------------------------
+; Subroutine to load pattern load cues (aka to queue pattern load requests)
 ; ---------------------------------------------------------------------------
-; Subroutines to load pattern load cues
 
-; input:
-;	d0 = pattern load cue number
+; ARGUMENTS
+; d0 = index of PLC lis
```

Comment:
typo: lis

Reply:
oops, gonna solve that as well as making the checksum comment clearer (as in add a note about it only being a signal for positives, not negatives)